src/win32/pthread/ptw32_callUserDestroyRoutines.c

```diff
@@ -164,9 +164,13 @@ ptw32_callUserDestroyRoutines (pthread_t thread)
 	       */
 	      k = assoc->key;
 	      destructor = k->destructor;
-	      value = TlsGetValue(k->key);
-	      TlsSetValue (k->key, NULL);
-
+#if REALM_UWP
+          value = FlsGetValue(k->key);
```

Comment:
It's in a 3'rd party .c file in which it uses a tab followed by spaces as default method in all the .c files in the directory - won't fix this.